README.md

```diff
@@ -1,3 +1,63 @@
+# Database Schema
+
```

Comment:
All of this is not required in the README file. The only thing required is how to connect to the database. Please remove all this schema definitions